psi4/share/psi4/fsapt/fsapt.py

```diff
@@ -836,6 +837,39 @@ def compute_fsapt(dirname, links5050, completeness = 0.85):
 
     osapt = extract_osapt_data(dirname)
 
+    # For I-SAPT/SAOn and I-SAPT/SIAOn, we need to add one extra orbital weight for the reassigned link orbital. It belongs
+    # to the atom of A/B directly connected to the linker C. The user needs to supply a file link_siao.dat that has two lines
+    #   A (atomnumber)
+    #   B (atomnumber)
+    # to specify the numbers of atoms which are connected to C. We will now check if this file exists and read it.
+    if os.path.exists("%s/link_siao.dat" % dirname):
+        (fragsiao,keyssiao) = read_fragments("%s/link_siao.dat" % dirname)
+        if ("A" not in keyssiao) or ("B" not in keyssiao):
+            print("\n Invalid syntax of the link_siao.dat file\n")
```

Comment:
Would it be cleaner to raise an exception here, instead of a print out? 

This is more of a formality than anything, since I think the code will throw on one of the next two lines anyway (depending on which of "A" or "B" is missing) if this line is printed.

Reply:
Good idea, fixed.